test/testdrive/kafka-avro-upsert-sinks.td

```diff
@@ -298,3 +298,95 @@ Invalid upsert key: (key1), valid keys are: (key1, key2)
   KEY (key2)
   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}' ENVELOPE UPSERT
 Invalid upsert key: (key2), valid keys are: (key1, key2)
+
```

Comment:
Not sure this should be here or in `kafka-upsert-sources.td`, to mirror the `new-syntax` file?